cfpb/teachers-digital-platform
# [TDP] Update CR Tool packages to fix broken Unit Tests

Second attempt at fixing CR Tool Unit tests. This time with fewer changes to the code base.

## Additions

- Updated node packages

## Changes

- Updated CR Tool's Jest Snapshots

## Testing

1. run ./setup.sh
2. run npm test
3. run tox -e lint
4. go to "practitioner-resources/youth-financial-education/curriculum-review/tool/" and make sure the app loads.

## Checklist
